pkg/controllers/secrets.go

```diff
@@ -74,8 +74,27 @@ func MapToEnvFormat(secrets map[string]string, wrapInQuotes bool) []string {
 	return env
 }
 
-func MapToDotNETFormat(secrets map[string]string) map[string]string {
-	return secrets
+func MapDotNETJSONFormat(secrets map[string]string) map[string]string {
+	var dotnetJSON = make(map[string]string)
+	var upperSnakeToUpperCamel = func(name string) string {
```

Comment:
minor: Could we move this function to `util.go` and perhaps include some tests in `util_test.go`? Potentially the whole `MapDotNETJSONFormat` could go in there, what do you think?

Reply:
@nmanoogian I like the idea of moving these to the utils package but put them in a `new name_transformers.go` file.